lua/conform/formatters/pymarkdownlnt.lua

```diff
@@ -0,0 +1,11 @@
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://github.com/jackdewinter/pymarkdown",
+    description = "PyMarkdown is primarily a Markdown linter",
+  },
+  command = "pymarkdownlnt",
+  args = { "--return-code-scheme", "minimal", "fix", "$FILENAME" },
+  exit_codes = { 0, 1 },
```

Comment:
Do we want this? According to the docs with `--return-code-scheme minimal` exit code 1 means `SYSTEM_ERROR`. I don't think that should be treated as a success.

Reply:
My apologies. I guess I did not understand that `exit_codes` refers only to the exit codes of successful execution, if this is what you're implying, but rather "possible exit codes you may encounter when running this command". I have removed the line entirely. Hope this is correct now?